projects/mac_ai/testing/scripts/check_podman_machine_status.sh

```
@@ -0,0 +1,29 @@
+
+SCRIPT_DIR=$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd)
+
+version=$(cat "$SCRIPT_DIR/src_info/version.txt")
+
+if [[ -z "$version" ]]; then
+  echo "Couldn't find the API Remoting version identifier :/"
+  exit 20
+fi
+
+# don't fail on this one ;-)
+LSOF_VIRGL=$((lsof -c krunkit || true) | (grep virglrenderer || true))
```

Shellcheck (0.10.0)

[error] Shells disambiguate $(( differently or not at all. For $(command substitution), add space after $( . For $((arithmetics)), fix parsing errors.

(SC1102)